apps/mudita-center-e2e/src/page-objects/overview-kompakt.page.ts

```diff
@@ -4,8 +4,52 @@
  */
 
 import { ChainablePromiseElement } from "webdriverio"
-import OverviewPage from "./overview.page"
+import { OverviewPage } from "./overview.page"
 
-class OverviewKompaktPage extends OverviewPage {}
+class OverviewKompaktPage extends OverviewPage {
+  public get aboutYourDevice() {
+    return $("p*=About your device")
+  }
 
+  public get kompaktImageElement() {
+    return $(
+      "#app > div.sc-hrCmsx.iRrrLX > div.sc-gtWJRm.fLIlvh > div.box-sizing-wrapper > div > div > div.sc-eGXPLf.bfaTWN > div > div.sc-eGXPLf.eAMdLd > img"
+    )
+  }
+
+  public get backupInfo() {
+    return $(`//div[@componentkey="backupbackup-info"]/p`)
+  }
+  public get serialNumberLabel() {
+    return $(`//div[@componentkey="summary-serial-number"]/p[1]`)
+  }
+
+  public get serialNumberValue() {
+    return $(`//div[@componentkey="summary-serial-number"]/p[2]`)
+  }
+
+  public get kompaktImage() {
+    return $('[data-testid="image"]')
+  }
+
+  public get kompaktSignalIcon() {
+    return $('[data-testid="icon-network-signal-2"]')
+  }
+
+  public get kompaktNetworkName() {
+    return $('[data-testid="icon-text"]')
+  }
+
+  public get kompaktBatteryLevel() {
+    return $('//div[@data-testid="icon-battery-charging-2"]')
```

Comment:
Its worth to split this into separate battery objects basing on charge percentage - you can use separate gets. or one function with switch to support all icons for battery. 

Your function / variable can't be called `kompaktBatteryLevel()` because it doesn't describe the level of battery level.

Reply:
Your function / variable can't be called kompaktBatteryLevel() because it doesn't describe the level of battery level. - changed to kompaktBatteryIcon

Reply:
current setup for battery level icon doesn't describe any state so it's universal:
public get kompaktBatteryLevelValue() {
    return $('//h3[text()="Status"]/..//h4[@data-testid="icon-text"]')
  }